http/sys_health.go

```diff
@@ -42,7 +42,9 @@ func fetchStatusCode(r *http.Request, field string) (int, bool, bool) {
 }
 
 func handleSysHealthGet(core *vault.Core, w http.ResponseWriter, r *http.Request) {
+	core.Logger().Debug("Checking system health")
```

Comment:
Is this leftover debugging, or do you want it to be part of your change?

Reply:
Unfortunately, this line is what makes all of my tests work 😂 so my preference would be to leave it in.

Reply:
I'll see if I can come up with a counter-proposal.  I see now where you depend on it, so I get your reasoning, but it would be nice if we didn't impact customer's logs.  Especially this endpoint which is likely to be hit all the time by monitoring systems.

Reply:
Sure, that sounds great.  FWIW, there's nothing special about this endpoint in particular.  It was an arbitrary choice.  I just wanted a way to generate log lines on demand, essentially.

Reply:
One thought that just occurred to me is that this is a `DEBUG` level log, and the default logging level is `INFO`, so out of the box, no additional log spamming will occur.  Customer logs would only be impacted if they enabled `DEBUG` level logging, and if they did, I would imagine they wouldn't be surprised to see lots of additional log spam.

I'm not sure if that helps this be less of a concern or not, but I figured it was a thought worth sharing.

Reply:
Many customers use DEBUG level logging.  How about you just mount and unmount a secrets engine to generate log traffic?  That's won't require prod code changes, and we're unlikely to ever break this because I can't see us deciding that doesn't warrant a log message in future.

Reply:
I agree with Nick's sentiment in here, `sys/health` is one of those endpoints that might be probed pretty frequently so this might end up spamming the server logs quite a bit when debug log level is set. Suggested a similar approach in one of the outdated comments:

> Another alternative to trigger a log line is to mount a secret under a random path. That way you won't need to add that debug log under handleSysHealthGet. You can look for [INFO] core: successful mount: so something along those lines.

Reply:
Sounds good.  I'll make that change.